pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -27,6 +27,8 @@ import (
 	"kubevirt.io/client-go/kubecli"
 	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 
```

Comment:
nit: empty line

Reply:
It's caused by `make generate` [1]. I'll try to adjust it.

[1] https://github.com/kubevirt/kubevirt/compare/732494c97a837f02112667dae2e8b55e0c1861b6..89ac73fe9d8f47f0ca591e9c85040f0ed5e7f006